benchmark.py

```diff
@@ -104,12 +104,34 @@ def main(job_config: JobConfig):
     )
 
     # apply PT-D Tensor Parallel, activation checkpointing, torch.compile, Data Parallel
-    model = torch_spmd_parallelize(model, world_mesh, parallel_dims, job_config)
-
+    if job_config.experimental.torch_spmd:
+        # apply PT-D Tensor Parallel, activation checkpointing, torch.compile, Data Parallel
+        model = torch_spmd_parallelize(model, world_mesh, parallel_dims, job_config)
+    else:
+       
```

Comment:
remove this empty line, and run `pre-commit` (install if you don't have it) after `git add` and before `git commit` -- it's similar to linting.